jbosstools/jbosstools-openshift
# JBIDE-25314 CDK launch configuration itests

- New test class CDKLaunchConfigurationTest
	- CDKSmoke and CDKAllTests suites extended with new test class
	- CDKLaunchConfigurationDialog added into cdk.reddeer plugin
	- New mock files representing cdk binaries
	- Remove deprecated CDK3 test classes
	- Add check of OS and docker after restart

Signed-off-by: Ondrej Dockal <odockal@redhat.com>

# Pull Request Checklist
## General
* Is this a blocking issue or new feature? If yes, QE needs to +1 this PR

## Code
* Are method-/class-/variable-names meaningful?
* Are methods concise, not too long?
* Are catch blocks catching precise Exceptions only (no catch all)?

## Testing
* Are there unit-tests?
* Are there integration tests (or at least a jira to tackle these)?
* Is the non-happy path working, too?
* Are other parts that use the same component still working fine?

## Function
* Does it work?
